retriever/lib/datasets.py

```diff
@@ -35,7 +35,8 @@ def datasets(keywords=None, licenses=None):
     result_scripts_offline = sorted(list(result_scripts_offline), key=lambda s: s.name.lower())
     result_retriever_scripts_online = get_dataset_names_upstream(keywords, licenses, repo=RETRIEVER_REPOSITORY)
     result_scripts_online = get_dataset_names_upstream(keywords, licenses)
-    return dict({'online': sorted(result_retriever_scripts_online + result_scripts_online), 'offline': result_scripts_offline})
+    return dict({'online': sorted(result_retriever_scripts_online + result_scripts_online), \
```

Comment:
The line length is still long for some line in this file. 
Please refactor them. Try to use autopep8 or black and manually edit as appropriate.
`autopep8 -i  -a retriever/lib/datasets.py `
or 
`autopep8 -i  -a  -a retriever/lib/datasets.py` # (very aggressive -a -a)
or 
`black retriever/lib/datasets.py`